src/test/store/receive-profile.test.js

```diff
@@ -629,40 +678,6 @@ describe('actions/receive-profile', function() {
         })
       );
     });
-
-    it('displays a warning after 30 seconds', async function() {
```

Comment:
Was this test not working anymore?

Reply:
Oh right, I meant to comment on that. So the problem here was that the test advances the time right after calling `retrieveProfileFromBrowser`. However, now that the web channel is checked first, there is an asynchronous gap before `retrieveProfileFromBrowser` calls `setTimeout`. So we advance the time before we call `setTimeout`, and consequently don't skip the timeout. I couldn't think of a way to work around that, other than firing something just after the call to setTimeout that the test could listen for.